controllers/movies.js

```diff
@@ -0,0 +1,48 @@
+const movies = require('../movies')
+
+const getAllMovies = (request, response) => {
+  return response.send(movies)
+}
+
+const getByTitleOrDirector = (request, response) => {
```

Comment:
_Nit_ function should return a 404 if the title or director doesnt exist. Not a 200